models/Settings.php

```diff
@@ -16,13 +14,17 @@ class Settings extends Model
     public $settingsCode = 'user_settings';
     public $settingsFields = 'fields.yaml';
 
+
     const ACTIVATE_AUTO = 'auto';
     const ACTIVATE_USER = 'user';
     const ACTIVATE_ADMIN = 'admin';
 
     const LOGIN_EMAIL = 'email';
     const LOGIN_USERNAME = 'username';
 
+    const MIN_PASSWORD_LENGTH = 'min_password_length';
```

Comment:
why is this a class constant? It seems unnecessary.

Reply:
No idea, it was in the original PR. I've removed it.